test/decorators/book_decorator_test.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class BookDecoratorTest < ActiveSupport::TestCase
+  setup do
+    @book1 = ActiveDecorator::Decorator.instance.decorate(books(:book1))
+    @book2 = ActiveDecorator::Decorator.instance.decorate(books(:book2))
+  end
+
+  test '#must_read_for_any_practices?' do
+    assert_equal true, @book1.must_read_for_any_practices?
+    assert_equal false, @book2.must_read_for_any_practices?
```

Comment:
```suggestion
    assert @book1.must_read_for_any_practices?
    assert_not @book2.must_read_for_any_practices?
```

trueかどうかのテストはassertがいいと思います〜

Reply:
修正しました！ご確認よろしくお願いいたします！